src/views/accounts/RiskDetect/RiskHandlerFormatter/const.js

```diff
@@ -9,7 +9,7 @@ async function checkAccountExist(username, asset) {
   }
   const url = `/api/v1/accounts/accounts/?username=${encodeURIComponent(username)}&asset=${asset}`
   const data = await this.$axios.get(url)
-  const results = data?.results
+  const results = data
   const isExist = results.length > 0
   accountExistCache.set(cacheKey, isExist)
   return isExist
```

Comment:
There do not appear to be any major syntax errors or issues with the provided code. No optimization suggestions can be made without specific details about the application of which the current environment may have additional constraints that could affect performance.

This query appears correct and will likely work in most environments but should it be used within an environment where there are different rules regarding cache usage, the implementation may need adjustments.